qstnrs/tests/conftest.py

```diff
@@ -0,0 +1,233 @@
+from questionnaire.models import Questionnaire, Page, Question, Answer
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def set_up(transactional_db):
+    # create Questionnaire
+    Questionnaire.objects.create(
+        questionnaire_name="Questionnaire_1",
+        questionnaire_description="Questionnaire 1"
+    )
+
+    Questionnaire.objects.create(
+        questionnaire_name="Questionnaire_2",
+        questionnaire_description="Questionnaire 2"
+    )
+
+    # create pages
+    Page.objects.create(
+        questionnaire_id=1,
```

Comment:
How can one know to which of the created questionnaire this page belongs to? since the id is auto-generated